src/commands.ts

```diff
@@ -148,22 +175,51 @@ const unsubCmd = new Command(
     if (!alias) {
       bot.sendMessage(
         channel,
-        `You need to provide the name of the game you want to unsubscribe from.\n`
-        + `Try ${unsubCmd.getTriggerLabel(channel)}.`,
+        `You need to provide the name of the game you want to unsubscribe from.\n` +
+          `Try ${unsubCmd.getTriggerLabel(channel)}.`,
       );
     }
 
-    for (const game of Game.getGames()) {
-      if (game.hasAlias(alias)) {
-        if (bot.removeSubscriber(channel, game)) {
-          bot.sendMessage(channel, `You are now unsubscribed from the **${game.label}** feed!`);
-        } else {
-          bot.sendMessage(
-            channel,
-            `You have never subscribed to the **${game.label}** feed in the first place!`,
-          );
+    const aliases = alias.split(', ');
+
+    // The games matching the alias
+    let aliasGames: Game[] = [];
+
+    for (alias of aliases) {
+      aliasGames = aliasGames.concat(Game.getGamesByAlias(alias));
+    }
+
+    if (aliasGames.length > 0) {
+      // The map of which game is a new sub
+      const gameMap = aliasGames.map((game) => {
+        const isNew = bot.removeSubscriber(channel, game);
+        return { game, isNew };
+      });
+
+      const newUnsubs = gameMap.filter((map) => map.isNew).map((map) => map.game);
+      const oldUnsubs = gameMap.filter((map) => !map.isNew).map((map) => map.game);
+
+      let message = '';
+
+      if (newUnsubs.length > 0) {
+        message += `You unsubscribed from ${naturalJoin(newUnsubs.map((game) => game.label))}.`;
+
+        if (oldUnsubs.length > 0) {
+          message +=  `\nYou have never subscribed to `
+          + `${naturalJoin(oldUnsubs.map((game) => game.label))} in the first place!`;
         }
+      } else {
```

Comment:
this looks unnecessary and over-complicated. Appending to the message based on array length should be enough

Reply:
The goal was to have the seperating linebreak, but in hindsight we can probably ommit the second if since Discord and Telegram trim leading whitespace anyway.

Reply:
Alright, I simplified the message composition.